src/core/persistence/Codend.Persistence/Configurations/SprintProjectTaskConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+using Codend.Domain.Entities;
+using Codend.Persistence.Extensions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Codend.Persistence.Configurations;
+
+public class SprintProjectTaskConfiguration : IEntityTypeConfiguration<SprintProjectTask>
+{
+    public void Configure(EntityTypeBuilder<SprintProjectTask> builder)
+    {
+        builder.ConfigureKeyId((Guid guid) => new SprintProjectTaskId(guid));
+        builder.ConfigureCreatableEntity();
+
+        builder
+            .HasOne<BaseProjectTask>()
```

Comment:
Not having some taskEntity  for tasks/epic/story to inherit from, made it impossible here to put story/epic into sprint. We have to do sth about this cuz board was supposed to be based on sprint, and we need both epics and stories there.